### PR TITLE
External UCS: ON by the User: Script 011

### PR DESCRIPTION
**Pull request by dboltovskyi**
Script is part of test set: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1301

Similar scripts:
https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1331
https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1332

### CODE REVIEW

File: test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    label = "label_test",
+    textBody = "textBody_test"
+  }
+  -- create json file for Policy Table Update
+  common_functions_external_consent:CreateJsonFileForPTU(data, "/tmp/ptu_update.json")
+  -- remove preload_pt from json file
+  local parent_item = {"policy_table","module_config"}
+  local removed_json_items = {"preloaded_pt"}
+  common_functions:RemoveItemsFromJsonFile("/tmp/ptu_update.json", parent_item, removed_json_items)
+  local removed_json_items_preloaded_date = {"preloaded_date"}
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @dboltovskyi, may be will be good to remove also 'preloaded_pt'.

File: test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+-- Prepare JSON file with consent groups. Add all consent group names into app_polices of applications
+-- Request Policy Table Update.
+--------------------------------------------------------------------------
+Test[TEST_NAME_ON.."Precondition_Update_Policy_Table"] = function(self)
+  -- create json for PTU from sdl_preloaded_pt.json
+  local data = common_functions_external_consent:ConvertPreloadedToJson()
+  -- insert Group001 into "functional_groupings"
+  data.policy_table.functional_groupings.Group001 = {
+    user_consent_prompt = "ConsentGroup001",
+    disallowed_by_external_consent_entities_off = {{
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @dboltovskyi, description of script is that disallowed_by_external_consent_entities_on exist, that may cause and update of some other actions.
Please update.

Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @istoimenova, Description is correct. 
According to req-t "[Policies] External UCS: "ON" - allowed RPCs"
SDL Policies database should contain "disallowed_by_external_consent_entities_off" param in "functional grouping" and then SDL gets SDL.OnAppPermissionConsent ("externalConsentStatus: ON")

File: test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua (hunk 1 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @istoimenova, It is removed in lines 105 - 106

File: test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua (hunk 2 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @istoimenova, Description of test script in test set will be updated

Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/011_ATF_Policies_External_Consent_ON_through_ignition_cycles_allowed_RPCs.lua]: @istoimenova,  Please find update in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1335

**Pull request merged**